src/_includes/banner.html

```diff
@@ -1,10 +1,7 @@
 <div class="banner">
   <p class="banner__text">
-    Dart 3 is here with 
-    <a href="https://dart.dev/language/records" class="no-automatic-external">records</a>, 
-    <a href="https://dart.dev/language/patterns" class="no-automatic-external">patterns</a>,
-    and <a href="https://dart.dev/language/class-modifiers" class="no-automatic-external">class modifiers</a>.
+    Dart 3.1 is out, with updates supporting some of the new features from Dart 3.0.
```

Comment:
Consider shortening this a bit.

```suggestion
    Dart 3.1 is out, with updates supporting the new features from Dart 3.0.
```